ironfish/src/wallet/wallet.ts

```diff
@@ -1510,15 +1510,20 @@ export class Wallet {
         }
 
         multisigKeys = {
-          keyPackage: accountValue.multisigKeys.keyPackage,
-          publicKeyPackage: accountValue.multisigKeys.publicKeyPackage,
+          ...accountValue.multisigKeys,
```

Comment:
same here as below so we don't lose any changes to multisigKeys someone else may make in the future. We should only use multisigKeys and not accountValue after line 1498.